addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1032,15 +1030,39 @@ def _channel_seen(self, last_message_id=None, allow_older=False):
         )
         if last_message_id is not False and not last_message:
             return
-        self._set_last_seen_message(last_message, allow_older=allow_older)
+        self._set_last_seen_message(last_message)
         return last_message.id
 
-    def _set_last_seen_message(self, last_message, allow_older=False, notify=True):
+
+    def _set_new_message_separator(self, message):
```

Comment:
As we're working with positional id rather than actual record relation, it's an exception where I would just give the id as param.